services/docker/webrecorder/nginx/nginx.conf

```diff
@@ -53,7 +53,8 @@ http {
     # Content Server
     server {
         listen 81;
-
+        listen 8092;
```

Comment:
Neat.

So, since it seems to be working locally and in Travis, I think we're good: this is not the nginx conf used in production, so a tweak won't affect any other environments.